test/e2e/e2e.bats

```diff
@@ -30,7 +30,7 @@ if ! skopeo inspect --creds "$SOURCE_AUTH_USERNAME:$SOURCE_AUTH_PASSWORD" "docke
 fi
 
 # Run the skopeo-copy task
-if ! tekton pipeline start skopeo-copy -p SOURCE_REGISTRY="$SOURCE_REGISTRY" \
+if ! tkn pipeline start skopeo-copy -p SOURCE_REGISTRY="$SOURCE_REGISTRY" \
```

Comment:
Please use a construction like this:
```bash
run false
assert_success
```
For instance: https://github.com/openshift-pipelines/task-git/blob/fbd3c8b15b10b0deda87538a8cbfb53aada6b1d3/test/e2e/e2e.bats#L11-L19